Resources/config/services.xml

```diff
@@ -185,10 +184,10 @@
             <argument type="constant">NULL</argument> <!-- expression evaluator -->
 
             <call method="setSerializationContextFactory">
-                <argument type="service" id="jms_serializer.serialization_context_factory" />
+                <argument type="service" id="jms_serializer.context_factory" />
```

Comment:
I will keep here two duplicate instances... having the same class, but two different services.

serialization_context_factory and deserialization_context_factory